manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DaringDevelopmentInc.Horizon
+PackageVersion: 2.9.0
+PackageLocale: en-US
+Publisher: Daring Development Inc.
+PackageName: Horizon
+License: Copyright (c) Daring Development Inc.. All rights reserved.
+ShortDescription: Horizon

```

Comment:
```
ShortDescription: Horizon
```
Please write a better description for what this application is about.

Reply:
Sorry, still really new to this, thought I had already put a better one in